utils.js

```diff
@@ -66,6 +66,11 @@ const getEventDescription = event => {
   return event.eventDescription
 }
 
+const forenameToInitial = name => {
+  if (!name) return null
+  return `${name.charAt()} ${name.split(' ').pop()}`
+}
```

Comment:
The name of this function is a bit weird but I can't think of anything better, maybe `shortenName`?! 
(could have a unit test)